R/indicator_native_range_year.R

```diff
@@ -0,0 +1,117 @@
+#' Create interactive plot for counts per native region and year of introduction
+#' 
+#' Based on \code{\link{countYearProvince}} plot from grofwild
+#' @param type character, native_range level of interest should be one of 
+#' \code{c("native_continent", "native_range")} 
+#' @param xlab character string, label of the x-axis. Defaults to "year".
+#' @param ylab character string, label of the y-axis. Defaults to "number of 
+#' alien species".  
+#' @inheritParams countYearProvince
+#' @return list with:
+#' \itemize{
+#' \item{'plot': }{plotly object, for a given specie the observed number 
+#' per year and per province is plotted in a stacked bar chart}
+#' \item{'data': }{data displayed in the plot, as data.frame with:
+#' \itemize{
+#' \item{'year': }{year at which the animal was introduced}
+#' \item{'nativeRange': }{native range of the introduced animal}
+#' \item{'aantal': }{counts of animals}
+#' }
+#' }
+#' }
+#' @import plotly
+#' @importFrom reshape2 melt
```

Comment:
As you use a function of `reshape2` you should add reshape2 to dependencies in `DESCRIPTION`. See comment below at lines 33-34 for more details.

Reply:
done